scenario_player/setup/__init__.py

```diff
@@ -0,0 +1,157 @@
+"""Scenario Player Setup Utilities.
+
+In order for the scenario player to run a scenario, it must first setup
+certain components, in order to be able to execute any assertions stated in a
```

Comment:
maybe `tasks` instead of `assertions`

Reply:
meet you halfway? `tasksertions`? :P
They're called `tasks` in the code, so that's better yea.